index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Currency Converter</title>
+  </head>
+  <body>
+    <!--Week1 Currency Converter!-->
+    <label for="currency">Create A New Rate</label>
+    <form id="currencyRateForm">
+      <input type="text" value="Danish Krone" id="baseCurrency" />
+      <input type="text" id="conjugateCurrency" />
```

Comment:
I think it would be better UX if input would have placeholder text, so its clear their purpose :)

Reply:
Okay, I did it thank you.